src/arkhamdb/DeckImporterMain.ttslua

```diff
@@ -391,6 +391,10 @@ end
 function handleCustomizableUpgrades(cardList, customizations)
   for _, card in ipairs(cardList) do
     if card.metadata.type == "UpgradeSheet" then
+      if customizations["isFormatted"] then
+        card.data["LuaScriptState"] = customizations[card.metadata.id]
+      end
```

Comment:
Needs comments to explain what it does

Reply:
This is specific to the deckbuilder, I'll just scrap it for now